README.md

```diff
@@ -1,52 +1,35 @@
-
-<b>Note:</b> Beta and Preview images are provided "as-is", "with all faults" and "as available" and are excluded from the service level agreement and warranty. Beta and Preview images may not be covered by customer support.
-
-***What images are available for GitHub Actions and Azure DevOps?***
-The availability of images for GitHub Actions and Azure DevOps is different. See documentation for more details:
-- [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources)
-- [Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software)
 
-***What image version is used in my build?*** Usually, image deployment takes 3-4 days, and documentation in the `main` branch is only updated when deployment is finished. To find out which image version and what software versions are used in a specific build, see `Set up job` (GitHub Actions) or `Initialize job` (Azure DevOps) step log.
+# GitHub Actions Runner Images
+**Table of Contents**
+- [About](#about)
+- [Available Images](#available-images)
+- [Image Definitions](#image-definitions)
+- [Image Releases](#image-releases)
+- [Software and Image Support](#software-and-image-support)
+- [How to Interact with the Repo](#how-to-interact-with-the-repo)
+- [FAQs](#faqs)
 
-***Looking for other Linux distributions?*** We do not plan to offer other Linux distributions. We recommend using Docker if you'd like to build using other distributions with the hosted virtual environments. Alternatively, you can leverage [self-hosted runners] and fully customize your environment to your needs.
 
-***How to contribute to macOS source?*** macOS source lives in this repository and available for everyone. However, MacOS image-generation CI doesn't support external contributions yet so we are not able to accept pull-requests for now.
-We are in the process of preparing MacOS CI to accept contributions. Until then, we appreciate your patience and ask you continue to make tool requests by filing issues.
 
-## Updates to virtual environments
-*Cadence*
-
-We typically deploy weekly updates to the software on the virtual environments.
-For some tools, we always install the latest at the time of the deployment; for others,
-we pin the tool to specific version(s).
+## About
+This repository contains the source code used to create the VM images for [GitHub Actions hosted runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners), as well as for [Microsoft-hosted agents](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops#use-a-microsoft-hosted-agent) used for Azure Pipelines.

```

Comment:
Do we state `GitHub Actions hosted runners` or `GitHub-hosted runners`?

Reply:
I will change for consistency